docs/Resonance_Compensation.md

```diff
@@ -45,31 +44,25 @@ Slice the ringing test model, which can be found in
 
 First, measure the **ringing frequency**.
 
-1. Increase `max_accel` and `max_accel_to_decel` parameters in your
-   `printer.cfg` to 7000. Note that this is only needed for tuning, and more
-   proper value will be selected in the corresponding
-   [section](#selecting-max_accel).
-2. If `square_corner_velocity` parameter was changed, revert it back to 5.0.
```

Comment:
Note that the way instructions go, it was suggested to set square_corner_velocity back to 5.0 and not change it back after tuning. If we remove this sentence, and instead suggest to set it via `SET_VELOCITY_LIMIT SQUARE_CORNER_VELOCITY=5`, then the testing will be done with one value, but the real printing will be done with a different value. This may be undesirable. So, maybe it makes sense to keep this point? Or mention it earlier, in the non-numbered part of the section. Alternatively, we should probably not suggest to reset SQUARE_CORNER_VELOCITY. However my estimations (via calibrate_shaper.py) indicate that having higher SQUARE_CORNER_VELOCITY very quickly increases the smoothing, such that SQUARE_CORNER_VELOCITY==10 will already create too much smoothing even with rather high resonance frequencies and low accelerations, this is why the suggestion was to keep SQUARE_CORNER_VELOCITY low.